Backend/Backend/Controllers/BookmarkController.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Backend.Controllers.Requests;
+using Backend.Models;
+using Backend.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Backend.Controllers;
+
+[ApiController]
+[Route("bookmark")]
+[Authorize]
+public class BookmarkController : ControllerBase
+{
+    private readonly AuthService _authService;
+    private readonly BookmarkService _bookmarkService;
+    private readonly ILogger<BookmarkController> _logger;
+    public BookmarkController(AuthService auth, BookmarkService bookmark, ILogger<BookmarkController> logger)
+    {
+        _authService = auth;
+        _bookmarkService = bookmark;
+        _logger = logger;
+    }
+
+    /// <summary>
+    /// Returns all bookmarks for logged in user for a given episode
+    /// </summary>
+    /// <param name="episodeId">Episode ID</param>
+    /// <returns></returns>
+    [HttpGet("{episodeId}/allBookmarks")]
+    public async Task<IActionResult> GetBookmarks([Required] Guid episodeId)
+    {
+        try
+        {
+            // Identify User from JWT Token
+            User user = (await _authService.IdentifyUserAsync(HttpContext))!;
+            return Ok(await _bookmarkService.GetBookmarks(user, episodeId));
```

Comment:
I know you put this in one liners everywhere, but would it be ok to two line this? Just for easier debugging since you can see the result before it fails

Reply:
this would just be for the ok